specs/dvc/replace_file_on_remote_storage.yml

```diff
@@ -0,0 +1,17 @@
+---
+name: Replace files on remote storage and/or an upstream repo
```

Comment:
@chrislemke similar to above, I think we could help indicate that it's a dvc workflow